struct.go

```diff
@@ -115,43 +122,133 @@ type K struct {
 	Data interface{}
 }
 
+func Bool(x bool) *K {
+	return &K{Type: -KB, Attr: NONE, Data: x}
```

Comment:
field names are not required here as all fields are listed.

Reply:
Sure, I will remove them